packages/sync-actions/src/product-types.js

```diff
@@ -67,7 +58,14 @@ export default (
     mapActionGroup,
     syncActionConfig
   )
-  const buildActions = createBuildActions(diffpatcher.diff, doMapActions)
+  const onBeforeApplyingDiff = null
+  const buildActions = createBuildActions(
+    diffpatcher.diff,
+    doMapActions,
+    onBeforeApplyingDiff,
+    { withHints: true }
```

Comment:
This could have been part of `syncActionConfig`. However that would mean _leaking_ the underlying API of `createBuildActions`  outwards